### DISCUSSION TITLE
Can i get a minimal Input section with a button to Right (little more organised) ?

### DISCUSSION THREAD

**Discussion by Sreenivas1323** [chakra-ui/chakra-ui#2436] [Q&A] [answered]
So, have been trying SSR with Next js and Chakra ui. Recently, i wanted a input of email and the button inside the input section, and while trying, i was always some OCD issues with the button placement. 
![Chakra issue](https://user-images.githubusercontent.com/50963808/98469870-24f43900-2208-11eb-845e-b6fac8456d17.PNG)
Can i get a better suggestions to get out of this?

code i used:

`        
           
<FormControl id="email">

          <FormLabel fontWeight="bold" >Email Id</FormLabel>
        <InputGroup size="lg" mt={2}>
        <Input
        fontWeight="semibold"
        pr="100px" 
        width="350px" 
        h="50px" 
        variant="filled"
          placeholder="ssaisreenivas@gmail.com"
          ref={inputEl}
          type="email"
          fontSize="md"
        />
        <InputRightElement  h="50px">
          <Button
            isLoading={loading}
            onClick={subscribe}
            width="150px" >
            Subscribe
          </Button>
        </InputRightElement>
      </InputGroup>

`

Suggestions or Improvements are massively appreciated!

**Comment by ljosberinn** [comment]
I'm not sure what your question is, can you clarify? Do you want less padding there? Do you want a suggestion on how to implement it? Do you just want feedback on how its looking in general?

**Reply by Sreenivas1323** [reply]
I want little less padding there, so that the button lining is more to its right matching the padding up and down.
Sorry for the late reply :(

**Comment by chasinhues** [comment, marked as answer]
I think this example in the docs should help in your case: https://chakra-ui.com/docs/form/input#password-input-example

Set `pr` on the `Input` to adjust it's padding.
